assets/src/carousel/index.js

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Glider from 'glider-js';
+import 'glider-js/glider.css';
+
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+
+domReady(() => {
+  const carouselWrappers = document.querySelectorAll(
+    '.web-stories-list__carousel'
+  );
+
+  const isRTL = 'rtl' === document.documentElement.getAttribute('dir');
+
+  if (!carouselWrappers.length) {
+    return;
+  }
+
+  const navArrows = {
+    prev: '.glider-prev',
+    next: '.glider-next',
+  };
+
+  // This is to add basic support for the RTL. We switch the nav arrows.
+  const navArrowsRTL = {
+    prev: '.glider-next',
+    next: '.glider-prev',
+  };
+
+  /**
+   * Override to add basic support for the nav arrows for RTL
+   *
+   * Glider-JS doesn't support RTL at the moment, this is to add basic
+   * functioning support for the nav arrows as otherwise the nav arrows
+   * becomes useless on RTL sites.
+   *
+   * @todo Maybe replace glider-js with other lightweight lib which has RTL support. or Replace it with 'amp-carousel' once we have the support.
+   *
+   * @param {Object|string} slide Slide arrow string based on action.
+   * @param {boolean}       dot   Is dot navigation action.
+   * @param {Object}        e     Event object.
+   *
+   * @return {boolean} Navigation done.
+   */
+  Glider.prototype.scrollItem = function (slide, dot, e) {
+    if (e) {
+      e.preventDefault();
+    }
+
+    const _ = this;
```

Comment:
The naming of this const is a little confusing. I thought it lodash was in use. Can we name this something cleaner.

Reply:
How about `const current = this`? or any other suggestion?

Reply:
Changed it to just `this` remove usage of `_` as any other words may not be able to define the usage exactly.

Reply:
Fine for me. This is just copied code that hopefully doesn't live long anyway. Don't need to change it too much.